tests/std/tests/P0009R18_mdspan_layout_stride/test.cpp

```diff
@@ -0,0 +1,150 @@
+        assert(cpy == m);
+        static_assert(is_nothrow_default_constructible_v<Mapping>);
+        static_assert(is_nothrow_copy_constructible_v<Mapping>);
+    }
+
+    { // Check construction from extents_type and array
+        Mapping m{ext, strs};
+        assert(m.extents() == ext);
+        assert(m.strides() == strs);
+        static_assert(is_nothrow_constructible_v<Mapping, Ext, Strides>);
+        // Other tests are defined in 'check_construction_from_extents_and_array' function [FIXME]
+    }
+
+    { // Check construction from extents_type and span
+        using Span = span<const IndexType, sizeof...(Extents)>;
+        Mapping m{ext, Span{strs}};
+        assert(m.extents() == ext);
+        assert(m.strides() == strs);
+        static_assert(is_nothrow_constructible_v<Mapping, Ext, Span>);
+        // Other tests are defined in 'check_construction_from_extents_and_array' function [FIXME]
+    }
+
+    using OtherIndexType = long long;
+    using Ext2           = extents<OtherIndexType, Extents...>;
+    using Mapping2       = layout_stride::mapping<Ext2>;
+
+    { // Check construction from other mappings
+        Mapping m1{ext, strs};
+        Mapping2 m2{m1};
+        assert(m1 == m2);
+        static_assert(is_nothrow_constructible_v<Mapping2, Mapping>);
+        // Other tests are defined in 'check_construction_from_other_mappings' function [FIXME]
+    }
+
+    Mapping m{ext, strs}; // For later use
+
+    { // Check 'extents' function
+        same_as<const Ext&> decltype(auto) ext2 = m.extents();
+        assert(ext2 == ext);
+        static_assert(noexcept(m.extents()));
+    }
+
+    { // Check 'strides' function
+        same_as<Strides> decltype(auto) strs2 = m.strides();
+        assert(strs2 == strs);
+        static_assert(noexcept(m.strides()));
+    }
+
+    // Function 'required_span_size' is tested in 'check_required_span_size' function[FIXME]
+    { // Check 'required_span_size' function[FIXME]
+        if (((ext.extent(Indices) == 0) || ...)) {
+            assert(m.required_span_size() == 0);
+        } else {
+            const IndexType expected_value =
+                static_cast<IndexType>((((ext.extent(Indices) - 1) * strs[Indices]) + ... + 1));
+            assert(m.required_span_size() == expected_value);
+        }
+        static_assert(noexcept(m.required_span_size()));
+    }
+
+    // Call operator() is tested in 'check_call_operator' function
+
+    { // Check 'is_always_[unique/exhaustive/strided]' functions
+        static_assert(Mapping::is_always_unique());
+        static_assert(!Mapping::is_always_exhaustive());
+        static_assert(Mapping::is_always_strided());
+    }
+
+    { // Check 'is_[unique/strided]' functions
+        static_assert(Mapping::is_unique());
+        static_assert(Mapping::is_strided());
+        // Tests of 'is_exhaustive' are defined in 'check_is_exhaustive' function [FIXME]
+    }
+
+    { // Check 'stride' function (intentionally not if constexpr)

```

Comment:
I'm confused by "(intentionally not if constexpr)" here - I don't see any `if` at all?

Reply:
`if` statement used to be here, I forgot to fix the comment